peepingtom/io_/reading/main.py

```diff
@@ -49,61 +50,59 @@ def read_file(file_path, **kwargs):
     raise ParseError(f'could not read {file_path}')
 
 
-def find_files(paths, filters=None, recursive=False, max=None):
+def expand_globs(globs, recursive=False, base_dir='.'):
     """
-    take a path or iterable thereof and find all the contained readable files
-    filters: a regex-like strings or iterable thereof used to match filenames
-    max: max number of files to read
+    expand globs to single files as appropriate for recursiveness; assume that
+    a directory as glob means "the contents of this directory"
```

Comment:
is this a dangerous assumption? might be worth having a kwarg to turn it off

Reply:
So, here are my thoughts: currently, `pt.read` can only read files individually. Though I plan to introduce at some point the ability to read a directory as a whole (e.g: cryosparc data is scattered through several files, and reading one file often makes no sense there), that will require some changes.
For example, we will have to add some functionality so that not only the file extension is used to detect what reader to use. Until then, a directory can only mean "every file inside here that can be read".

Reply:
I changed this top be explicit in the end. Code is easier, and things might have behaved weirdly before!